inc/Engine/Media/AboveTheFold/AJAX/Controller.php

```diff
@@ -212,4 +217,42 @@ public function check_lcp_data() {
 
 		wp_send_json_error( 'data does not exist' );
 	}
+
+	/**
+	 * Validate image object.
+	 *
+	 * @param object $image_object Image full object.
+	 * @return bool
+	 */
+	private function validate_image( $image_object ) {
+		/**
+		 * Filters If the image src is a valid image or not.
+		 *
+		 * @param bool   $valid_image Valid image or not.
+		 * @param string $image_src_url Image src url.
+		 * @param object $image_object Image object with full details.
+		 */
+		return (bool) apply_filters( 'rocket_atf_valid_image', $this->validate_image_src( $image_object->src ?? '' ), $image_object->src, $image_object );
```

Comment:
suggesting for readability to assign the value of the method to variable `$valid_image`, and use the variable into the filter

Reply:
Fixed :)